plugins/github-actions/src/components/WorkflowRunsTable/useWorkflowRuns.ts

```diff
@@ -15,42 +15,58 @@
  */
 import { useState } from 'react';
 import { useAsyncRetry } from 'react-use';
-import { Build } from './BuildListTable';
+import { WorkflowRun } from './WorkflowRunsTable';
 import { githubActionsApiRef } from '../../api/GithubActionsApi';
-import { useApi, githubAuthApiRef } from '@backstage/core';
+import { useApi, githubAuthApiRef, errorApiRef } from '@backstage/core';
 import { ActionsListWorkflowRunsForRepoResponseData } from '@octokit/types';
 
-export function useBuilds({ repo, owner }: { repo: string; owner: string }) {
+export function useWorkflowRuns({
+  repo,
+  owner,
+}: {
+  repo: string;
+  owner: string;
+}) {
   const api = useApi(githubActionsApiRef);
   const auth = useApi(githubAuthApiRef);
 
+  const errorApi = useApi(errorApiRef);
+
   const [total, setTotal] = useState(0);
   const [page, setPage] = useState(0);
   const [pageSize, setPageSize] = useState(5);
 
-  const restartBuild = async () => {};
+  const reRunWorkflow = async () => {};
```

Comment:
Skipping this for now?

Reply:
I've implemented it buut there are some troubles with access rights. I'm trying to figure out how to fix it.

Reply:
I'm mistaken. I don't really understand where this one should be used. We already have rerun workflow button on each row of the table.